[0m19:45:20.597672 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D8990D87D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D898A92C10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D898C0FF90>]}


============================== 19:45:20.603614 | 4b669612-f850-4d9d-b179-6b2b44e80c5c ==============================
[0m19:45:20.603614 [info ] [MainThread]: Running with dbt=1.6.6
[0m19:45:20.605610 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\aadit\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt init', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m19:45:20.608535 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/aadit/.dbt')}
[0m19:45:20.610790 [info ] [MainThread]: Creating dbt configuration folder at 
[0m19:48:24.619141 [debug] [MainThread]: Starter project path: C:\Users\aadit\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m19:48:24.745492 [info ] [MainThread]: 
Your new dbt project "bde" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m19:48:24.750443 [info ] [MainThread]: Setting up your profile.
[0m19:49:48.346848 [info ] [MainThread]: Profile bde written to C:\Users\aadit\.dbt\profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m19:49:48.349841 [debug] [MainThread]: Command `dbt init` succeeded at 19:49:48.349841 after 267.77 seconds
[0m19:49:48.350837 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D898913550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D892168510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D89216B790>]}
[0m19:49:48.352139 [debug] [MainThread]: Flushing usage events
[0m16:20:22.907592 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026742C7FF50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000267427A47D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026742512410>]}


============================== 16:20:22.973752 | 29eca5a5-8312-4816-a053-801f2b7049f4 ==============================
[0m16:20:22.973752 [info ] [MainThread]: Running with dbt=1.6.6
[0m16:20:22.976712 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\aadit\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m16:20:22.978708 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\aadit\Desktop\UTS\4. Spring 2023\2. Big Data Engineering\Assignments\3. Building ELT data pipelines with Airflow\dbt-dev\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:20:22.982695 [debug] [MainThread]: Command `dbt run` failed at 16:20:22.982695 after 0.14 seconds
[0m16:20:22.983696 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026742437F10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002673BD78890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000267427A4290>]}
[0m16:20:22.985688 [debug] [MainThread]: Flushing usage events
[0m18:58:13.817982 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B7F06950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B78C79D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B7E27010>]}


============================== 18:58:13.823555 | 36c76909-5527-4cc3-9c3e-855eb65cb5b5 ==============================
[0m18:58:13.823555 [info ] [MainThread]: Running with dbt=1.6.6
[0m18:58:13.834696 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\aadit\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m18:58:13.835694 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\aadit\Desktop\UTS\4. Spring 2023\2. Big Data Engineering\Assignments\3. Building ELT data pipelines with Airflow\dbt-dev\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:58:13.837717 [debug] [MainThread]: Command `dbt run` failed at 18:58:13.837717 after 0.07 seconds
[0m18:58:13.838744 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B7EAEA90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B1292450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001D4B1528510>]}
[0m18:58:13.840149 [debug] [MainThread]: Flushing usage events
[0m19:03:43.651050 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002043974BA50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020439827950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020439C4CD90>]}


============================== 19:03:43.658035 | a25c010a-45dc-4434-8851-6b468eca9ff9 ==============================
[0m19:03:43.658035 [info ] [MainThread]: Running with dbt=1.6.6
[0m19:03:43.660295 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\aadit\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m19:03:43.662024 [info ] [MainThread]: dbt version: 1.6.6
[0m19:03:43.662601 [info ] [MainThread]: python version: 3.11.4
[0m19:03:43.663710 [info ] [MainThread]: python path: C:\Users\aadit\AppData\Local\Programs\Python\Python311\python.exe
[0m19:03:43.665172 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m19:03:43.758857 [info ] [MainThread]: Using profiles dir at C:\Users\aadit\.dbt
[0m19:03:43.760083 [info ] [MainThread]: Using profiles.yml file at C:\Users\aadit\.dbt\profiles.yml
[0m19:03:43.761819 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\aadit\Desktop\UTS\4. Spring 2023\2. Big Data Engineering\Assignments\3. Building ELT data pipelines with Airflow\dbt-dev\dbt_project.yml
[0m19:03:43.763844 [info ] [MainThread]: adapter type: postgres
[0m19:03:43.764817 [info ] [MainThread]: adapter version: 1.6.6
[0m19:03:43.766814 [info ] [MainThread]: Configuration:
[0m19:03:43.768801 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m19:03:43.769937 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m19:03:43.771794 [info ] [MainThread]: Required dependencies:
[0m19:03:43.772933 [debug] [MainThread]: Executing "git --help"
[0m19:03:43.852467 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m19:03:43.853497 [debug] [MainThread]: STDERR: "b''"
[0m19:03:43.854229 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m19:03:43.855528 [info ] [MainThread]: Connection:
[0m19:03:43.856885 [info ] [MainThread]:   host: 34.116.125.98
[0m19:03:43.858256 [info ] [MainThread]:   port: 5432
[0m19:03:43.860251 [info ] [MainThread]:   user: postgres
[0m19:03:43.861311 [info ] [MainThread]:   database: postgres
[0m19:03:43.863601 [info ] [MainThread]:   schema: raw_schema
[0m19:03:43.865434 [info ] [MainThread]:   connect_timeout: 10
[0m19:03:43.867793 [info ] [MainThread]:   role: None
[0m19:03:43.868791 [info ] [MainThread]:   search_path: None
[0m19:03:43.870001 [info ] [MainThread]:   keepalives_idle: 0
[0m19:03:43.872886 [info ] [MainThread]:   sslmode: None
[0m19:03:43.874006 [info ] [MainThread]:   sslcert: None
[0m19:03:43.875500 [info ] [MainThread]:   sslkey: None
[0m19:03:43.876791 [info ] [MainThread]:   sslrootcert: None
[0m19:03:43.878441 [info ] [MainThread]:   application_name: dbt
[0m19:03:43.880303 [info ] [MainThread]:   retries: 1
[0m19:03:43.882915 [info ] [MainThread]: Registered adapter: postgres=1.6.6
[0m19:03:43.884049 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m19:03:43.886075 [debug] [MainThread]: Using postgres connection "debug"
[0m19:03:43.886955 [debug] [MainThread]: On debug: select 1 as id
[0m19:03:43.888958 [debug] [MainThread]: Opening a new connection, currently in state init
[0m19:03:43.969448 [debug] [MainThread]: SQL status: SELECT 1 in 0.0 seconds
[0m19:03:43.973484 [debug] [MainThread]: On debug: Close
[0m19:03:43.975477 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m19:03:43.979430 [info ] [MainThread]: [31m1 check failed:[0m
[0m19:03:43.984417 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\aadit\Desktop\UTS\4. Spring 2023\2. Big Data Engineering\Assignments\3. Building ELT data pipelines with Airflow\dbt-dev\dbt_project.yml> not found

[0m19:03:43.991397 [debug] [MainThread]: Command `dbt debug` failed at 19:03:43.990400 after 0.36 seconds
[0m19:03:43.992395 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m19:03:43.995081 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020432AF2490>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020438F44AD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000204391EF3D0>]}
[0m19:03:43.997378 [debug] [MainThread]: Flushing usage events
[0m20:11:18.236611 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC88AEA50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC8D70090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC862B2D0>]}


============================== 20:11:18.252263 | f81153bc-485b-4b67-8b18-ad7ea726287e ==============================
[0m20:11:18.252263 [info ] [MainThread]: Running with dbt=1.6.6
[0m20:11:18.252263 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\aadit\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run --full-refresh', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:11:18.252263 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\aadit\Desktop\UTS\4. Spring 2023\2. Big Data Engineering\Assignments\3. Building ELT data pipelines with Airflow\dbt-dev\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:11:18.262279 [debug] [MainThread]: Command `dbt run` failed at 20:11:18.261282 after 0.04 seconds
[0m20:11:18.263277 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC88AD110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC89443D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BAC8606B90>]}
[0m20:11:18.264275 [debug] [MainThread]: Flushing usage events
